pandas/tests/groupby/test_any_all.py

```diff
@@ -83,6 +83,25 @@ def test_any_non_keyword_deprecation():
     tm.assert_equal(result, expected)
 
 
+def test_any_apply_keyword_non_zero_axis_regression():
+    # https://github.com/pandas-dev/pandas/issues/48656
```

Comment:
Apologies, I should've commented on this earlier, but I don't think this the right location for this test, as there's no `groupby` in this test

Let's put it in

https://github.com/pandas-dev/pandas/blob/221f6362bc25833da87f00015d4d5418ee316eff/pandas/tests/apply/test_frame_apply.py

(admittedly the test above, `test_any_non_keyword_deprecation`, is also in the wrong location 😳 I'll open a separate issue about that)